pkg/datamodel-tests/src/implementations/version-1.1/db.ts

```diff
@@ -17,6 +17,25 @@ class Database implements DatabaseInterface {
 		// All of this is done automatically when running db.warehouse('0-all')
 		this.warehouse('0-all').create();
 	}
+	async init(remote?: { database: PouchDB.Database }) {
```

Comment:
I know this makes sense to you when writing a function, but think about it also making sense when calling it: Imagine you're calling this `db.init({ database })` -> it's kinda confusing...rename it so that it communicates that it's remote db you're passing in.

Reply:
You can just name it `params` and have a property `remoteDb` (or just `remote`, like you have in the helper function)